pkg/rancher-desktop/backend/backendHelper.ts

```diff
@@ -278,9 +278,9 @@ export default class BackendHelper {
 
       await vmx.execCommand({ root: true }, 'mkdir', '-p', path.dirname(MANIFESTS_RUNTIMES_YAML));
       await vmx.writeFile(MANIFESTS_RUNTIMES_YAML, manifest, 0o644);
-      // Don't let k3s define runtime classes, only use the ones defined by Rancher Desktop.
-      await vmx.execCommand({ root: true }, 'touch', `${ path.dirname(MANIFESTS_RUNTIMES_YAML) }/runtimes.yaml.skip`);
     }
+    // Don't let k3s define runtime classes, only use the ones defined by Rancher Desktop.
+    await vmx.execCommand({ root: true }, 'touch', `${ path.dirname(MANIFESTS_RUNTIMES_YAML) }/runtimes.yaml.skip`);
```

Comment:
Is `path.dirname(MANIFESTS_RUNTIMES_YAML)` guaranteed to exist? (We don't run `mkdir -p` in the no runtimes case)

Reply:
Good point, updated.